### ISSUE TITLE
[BUG] Clear Button Window Disapearing Before Being Interacted With

### ISSUE THREAD

**Issue by josh-justjosh** [bitfocus/companion#3042]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

_For clarity in this report:_
- _I use "Companion Button" to mean the programmable buttons in the grid that appear on streamdecks_
- _I use "Control Button" to mean buttons within the GUI to perform actions on the Companion Buttons_
---
When trying to delete a companion button in v3.4.0 using the delete control button, when clicking on the companion button I want to delete, the clear button window only appears while I am holding down left-click and disappears when I release the mouse. This makes it impossible to delete companion buttons without using the keyboard.

### Steps To Reproduce

1. Navigate to the buttons view in the GUI
2. Click the delete control button
3. Click a companion button to clear
4. Obverve the clear button window only remaining visible while holding down left-click

### Expected Behavior

I'd expect the clear button window to remain visible until either the Clear, Cancel or X buttons are clicked on the window or I have clicked somewhere in the GUI outside the window

### Environment (please complete the following information)

```markdown
- OS: Windows 10 Pro 22H2 - build 19045.4123
- Browser: Chrome 128.0.6613.138 (Official Build) (64-bit) & 129.0.6668.59 (Official Build) (64-bit) _(Chrome updated while I was looking up the version number)_
- Companion Version: 3.4.0+7304-stable-01df0f17
```

### Additional context

I tried this both with my normal companion configuration and a completely blank default config and got the same result

**Comment by mou22y2** [comment]
I got the exact same problem since 3.4.0 and tried 3.4.1 today. And the problem with deleting a button is still there. 
It is possible to press delete on keyboard but not using the delete in UI as you got used to when copying, deleteing and moving buttons..

**Comment by thedist** [comment]
You can still delete within the UI, but it requires clicking and holding, moving the cursor into the modal and then you can release the mouse button and click on a on buttons within it. It's an annoying workaround, so hopefully this will be resolved in a patch soon.

**Comment by Julusian** [comment]
This is fixed in the v3.4.2 release
